tests/test_utils.py

```diff
@@ -121,3 +121,27 @@ class Rect(utils.Function):
     obj = Rect('"hello world"')
     assert obj.render() == '#rect("hello world", width: 10em, height: 10%, fill: red)'
     test_compile(obj)
+
+
+def test_empty_string():
+    obj = utils.String()
+    assert obj.render() == '#""'
+
+
+def test_escaped_string():
+    line = "The most 'common' string in programming is: " + '"Hello world!".'
+    s = utils.String(line)
+    assert (
+        s.render()
+        == '#"The most \'common\' string in programming is: \\"Hello world!\\"."'
+    )
+
+
+@pytest.mark.integration
+def test_escaped_string_compile(test_compile):
+    line = "The most 'common' string in programming is: " + '"Hello world!".'
+    s = utils.String(line)
+    let = f"#let foo = {utils.render_code(s)}"
+    check = '#assert.eq(foo, "The most \'common\' string in programming is: \\"Hello world!\\".")'
```

Comment:
Trippy assertion across languages. I like it 😁

Reply:
Thought it made the most sense to truly test the usage of a string variable.

At such times, strings reveal themselves to be the rabbit hole they are :rabbit: